src/window/Window.cpp

```diff
@@ -14,6 +14,11 @@
 #include "util/ObjectsKeeper.hpp"
 #include "Events.hpp"
 
+#define NOMINMAX
+#include <Windows.h>
```

Comment:
?

Reply:
Leftovers from the initial implementation. Removed in the following commit.